back/engines/commercial/report_builder/app/policies/report_builder/live_graph_data_unit_policy.rb

```diff
@@ -0,0 +1,12 @@
+module ReportBuilder
+  class LiveGraphDataUnitPolicy < ::ApplicationPolicy
+    def live?
+      props = record
+      if props[:project_id].present?
```

Comment:
I don't know if this is much more difficult, but if we move this check into `analytics` we can also make the analytics endpoint available for project moderators under some conditions. But I assume it's much more difficult with the whole query language

Reply:
Yes, this check will be different for different fact tables. We can do it later though.